src/database/queries/reviews.js

```diff
@@ -14,19 +14,27 @@ exports.findAll = async (filter, attributes = {}) => {
 		return error
 	}
 }
-exports.countDistinct = async (filter, attributes = {}) => {
+
+exports.countDistinct = async (filter, attributes = ['resource_id']) => {
 	try {
-		const res = await reviews.count({
+		const res = await reviews.findAndCountAll({
 			where: filter,
-			distinct: true,
+			attributes: attributes, // Select the specified attributes, defaulting to 'resource_id'
+			distinct: true, // Ensure distinct results
 		})
 
-		return res
+		// Extract unique resource IDs
+		const uniqueResourceIds = res.rows.map((row) => row.resource_id)
+
+		return {
+			count: res.count,
+			resource_ids: uniqueResourceIds,
+		}
```

Comment:
@adithyadinesh0412 if duplicates are there then map you can do here
resource_ids: res.rows.map((row) => row.resource_id)